openapi.yaml

```diff
@@ -6726,6 +6726,48 @@ paths:
       - lang: CLI
         source: >
           linode-cli lke pool-delete 12345 456
+  /lke/clusters/{clusterId}/pools/{poolId}/recycle:
+    parameters:
+    - name: clusterId
+      in: path
+      description: ID of the Kubernetes cluster to this Node Pool is attached to.
+      required: true
+      schema:
+        type: integer
+    - name: poolId
+      in: path
+      description: ID of the Kubernetes pool to be recycled.
+      required: true
+      schema:
+        type: integer
+    x-linode-cli-command: lke
+    post:
+      operationId: postLKEClusterPoolsRecycle
+      x-linode-cli-action: cluster-recycle
```

Comment:
same as below, can we change this to "pool-recycle"? might be a question for the CLI team

Reply:
updated